### PR TITLE
feat(api): improve validation pipeline for representation routes

### PR DESCRIPTION
**Pull request by fabfab-kc**
Removed redundant validation from rep routes
Improved validation messages

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

**Review by kandcdev — APPROVED** [review]

File: appeals/api/src/server/endpoints/constants.js [hunk 1]
```diff
@@ -156,7 +158,7 @@ export const ERROR_INVALID_LPAQ_DATA = 'The integration payload LPA_QUESTIONNAIR
 export const ERROR_INVALID_DOCUMENT_DATA = 'The integration payload DOCUMENT is invalid.';
 export const ERROR_INVALID_APPEAL_STATE = 'The action is invalid on the current appeal state.';
 export const ERROR_CASE_OUTCOME_MUST_BE_ONE_OF = `The case outcome must be one of ${CASE_OUTCOME_ALLOWED}, ${CASE_OUTCOME_DISMISSED}, ${CASE_OUTCOME_SPLIT_DECISION}`;
-export const ERROR_REP_OUTCOME_MUST_BE_ONE_OF = `The representation status must be valid`;
+export const ERROR_REP_OUTCOME_MUST_BE_ONE_OF = `The representation status must be on of ${APPEAL_REPRESENTATION_STATUS.AWAITING_REVIEW}, ${APPEAL_REPRESENTATION_STATUS.INVALID}, ${APPEAL_REPRESENTATION_STATUS.VALID}`;
```
Comment (james-hill-kc) [inline appeals/api/src/server/endpoints/constants.js:161]: Minor, typo here - `on` should be `one`

File: appeals/api/src/server/endpoints/representations/__tests__/representations.test.js [hunk 2]
```diff
@@ -83,7 +83,7 @@ describe('/appeals/:id/representations', () => {
 			expect(response.status).toEqual(400);
 			expect(response.body).toEqual({
 				errors: {
-					status: 'The representation status must be valid'
+					status: 'The representation status must be on of awaiting_review, invalid, valid'
```
Comment (james-hill-kc) [inline appeals/api/src/server/endpoints/representations/__tests__/representations.test.js:86]: Same typo here, `on` should be `one`

**Review by james-hill-kc — COMMENTED** [review]
Looks good, just spotted a typo

**Review by james-hill-kc — APPROVED** [review]
LGTM!

**Pull request merged**